### ISSUE TITLE
infer tails and infer heads for ConvE

### ISSUE THREAD

**Issue by saatviga-healx** [Sujit-O/pykg2vec#81]
Hi,

Can you include the infer tails and infer heads functionality of the ConvE model? Its not there but I saw its available for TransE. I get an error when using the reshaping and batch normalisation for the inference provided in the test method. It works fine for testing a batch, but when given individual pairs like (head,rel) or (tail,rel) it does not work.

The error I get is,
cuDNN launch failure : input shape ([1,1,20,20])
         [[node ConvEModel/ConvE/batch_normalization_v1/cond/FusedBatchNorm_1

**Comment by baxtree** [comment]
Expected functionality was added by https://github.com/Sujit-O/pykg2vec/pull/88.
